workflow-templates/library-checks-and-release.yml

```diff
@@ -0,0 +1,57 @@
+name: Library checks & release
+
+on:
+  push:
+    branches:
+      - master
+      - next
+      - alpha
+      - beta
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  PULL_REQUEST_NUMBER: ${{ github.event.pull_request.number }}
+  SURGE_LOGIN: ${{ secrets.SURGE_LOGIN }}
+  SURGE_TOKEN: ${{ secrets.SURGE_TOKEN }}
```

Comment:
Are `SURGE_LOGIN`, `SURGE_TOKEN ` and `PULL_REQUEST_NUMBER` necessary? There aren't used in the rest of the template

Reply:
No, No and No. They're only required to generate storybook preview links for very specific libraries (Design System), so I'd remove it from the template.